runtime/lua/vim/lsp/util.lua

```diff
@@ -782,11 +803,14 @@ function M._make_floating_popup_size(contents, opts)
       width = math.max(line_widths[i], width)
     end
   end
+  if max_width then
+    width = math.min(width, max_width)
+    wrap_at = math.min(wrap_at, max_width)
```

Comment:
can this fail if `wrap_at` is nil? I don't remember how math.min works for that.

Reply:
It works like this: (╯°□°)╯︵ ┻━┻

(I don't think that happens in the current code path, but it should be made robust nevertheless.)

Reply:
Yeah, just checked and it fails for nil values.

Reply:
so maybe -- `wrap_at = wrap_at and math.min(wrap_at, max_width)` like that.

Reply:
there, I fixed it :)

Reply:
(I think if you explicitly set a max width, you also want to wrap at that width, so I opted for `wrap_at = math.min(wrap_at or max_width, max_width)`.)